python/instrumentation/openinference-instrumentation-litellm/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+## 0.1.0 (2024-07-29)
+
+### Features
+
+* Added liteLLM instrumentation for the following functions:
+    - completion()
+    - acompletion()
+    - completion_with_retries()
+    - acompletion_with_retries() **
```

Comment:
We should put `acompletion_with_retries` into a separate section for methods that are planned but currently not working

Reply:
Done, put in separate section